ansible/roles/webserver-content/tasks/main.yml

```diff
@@ -20,33 +20,13 @@
   git:
     repo: "{{ connectbox_client_repo }}"
     dest: /tmp/connectbox_client
+    version: react_admin_ui
```

Comment:
This will need to be removed before merging.

Reply:
That's correct. This is just there so that this will pull the branch version of the UI until I release the it to master.